test/e2e/validation_test.go

```diff
@@ -0,0 +1,47 @@
+// +build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/argoproj/argo/test/util"
+)
+
+func TestValidateExamples(t *testing.T) {
+	failures, err := util.ValidateArgoYamlRecursively("../../examples", []string{"testvolume.yaml"})
```

Comment:
this should probably be in the `../examples` directory

Reply:
Sure, just confirm you definitely don't want to check any of the e2e workflow manifests for validity and I'll see what I can do.

Reply:
done